MLH-Fellowship/docusaurus
# feat(v2): add a badge that links to latest version

## Motivation

This PR adds a badge that links older versions of documentation to the latest version of the documentation.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan 
1. Go to any older version in the preview site. Check that there is a badge that links to the latest version
![image](https://user-images.githubusercontent.com/46853051/84246362-1b6f8380-ab39-11ea-96da-ffb5e8718c17.png)
1. Click on the link to the latest version. There should be no badge next to the version number, since we are now at the latest version
![image](https://user-images.githubusercontent.com/46853051/84246449-39d57f00-ab39-11ea-97bd-78e59cd4dbf4.png)

Preview: https://deploy-preview-2915--docusaurus-2.netlify.app
